demo/api/src/site-configs.d.ts

```diff
@@ -0,0 +1,18 @@
+import type { BaseSiteConfig, ExtractPrivateSiteConfig, ExtractPublicSiteConfig } from "@comet/cli";
+
+interface ContentScope {
+    domain: string;
+    language: string;
+}
+
+export interface SiteConfig extends BaseSiteConfig {
+    public: {
+        scope: {
+            domain: string;
+            languages: string[];
+        };
+    };
+}
+
+export type PrivateSiteConfig = ExtractPrivateSiteConfig<SiteConfig>;
+export type PublicSiteConfig = ExtractPublicSiteConfig<SiteConfig>;
```

Comment:
this file should be a symlink to demo/site-configs/site-configs.d.ts, not a copy

Reply:
no, it is not. The copy-schema-files.js you linked is executed in pipelines only.

The file should not be duplicated in the git repository, with the need to apply any changes 3 times.